code/modules/power/cable.dm

```diff
@@ -626,33 +631,20 @@ obj/structure/cable/proc/cableColor(var/colorC)
 	if(!selected_color)
 		return
 
-	our_color = selected_color
-	if(!our_color)
-		our_color = "Red"
-	update_icon()
+	var/final_color = possible_cable_coil_colours[selected_color]
+	if(!final_color)
+		final_color = possible_cable_coil_colours["Red"]
```

Comment:
shouldn't you just be pick()ing one here

Reply:
this is the proc for changing cable colors using the multitool.

it doesn't pick because here it's supposed to go to default red